galleries/examples/subplots_axes_and_figures/two_scales.py

```diff
@@ -49,3 +49,9 @@
 #    - `matplotlib.axes.Axes.twinx` / `matplotlib.pyplot.twinx`
 #    - `matplotlib.axes.Axes.twiny` / `matplotlib.pyplot.twiny`
 #    - `matplotlib.axes.Axes.tick_params` / `matplotlib.pyplot.tick_params`
+#
+# .. tags::
+#
+#    component: Axes
```

Comment:
It could be failing because this is `Axes` while the others are `axes`:
> /home/circleci/project/doc/_tags/tagsindex.rst:8: WARNING: duplicated entry found in toctree: _tags/component-axes

Reply:
That did it! Thanks for finding it.